lib/modules/versioning/types.ts

```diff
@@ -10,26 +10,88 @@ export interface NewValueConfig {
 }
 export interface VersioningApi {
   // validation
+
+  /**
+   * Check whether the `version` is compatible with the `current` version range
```

Comment:
```suggestion
   * Check whether the `version` is compatible with the `current` version
```
🤔 can probably a version or range.

Reply:
hmm, not sure I understand this 😅

Reply:
updated my suggestion. we use this to check if the new version is compatible with the current extracted value. eg for docker versioning the suffix like `-alpine` needs to match, so we don't update a docker images from alpine to debian.